pkg/apis/deviceplugin/v1/qatdeviceplugin_webhook.go

```diff
@@ -88,17 +89,18 @@ func (r *QatDevicePlugin) ValidateDelete() error {
 }
 
 func (r *QatDevicePlugin) validatePlugin() error {
-	parts := strings.SplitN(r.Spec.Image, ":", 2)
```

Comment:
```suggestion
return validatePluginImage(r.Spec.Image, "intel-qat-plugin", qatMinVersion)
```

we could just return the output from `validatePluginImage` (see, e.g., dsadeviceplugin_webhook.go)

Reply:
Thanks for the input. Done. Please take a look.